src/components/ChangePlan/CheckoutProcess.test.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { render, cleanup, waitFor, getByTitle } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import IntlProvider from '../../i18n/DopplerIntlProvider.double-with-ids-as-values';
+import { AppServicesProvider } from '../../services/pure-di';
+import { BrowserRouter } from 'react-router-dom';
+import CheckoutProcess from './CheckoutProcess';
+
+describe('CheckoutProcess component', () => {
+  afterEach(cleanup);
+
+  it('should show contact information', async () => {
+    // Arrange
+    const contactInformation = {
+      email: 'test@makingsense.com',
+      firstname: 'Juan',
+      lastname: 'Perez',
+      address: 'Alem 1234',
+      city: 'Tandil',
+      province: 'Buenos Aires',
+      countryId: 1,
+      zipCode: '7000',
+      phoneNumber: '+5424966666',
+      company: 'Test',
+      industry: 'IT',
+      isCompleted: true,
```

Comment:
@cbernat should it be `completed`? right?

Reply:
yes, i changed it.